src/common/util/navigator_gpu.ts

```diff
@@ -43,6 +43,9 @@ export function setDefaultRequestAdapterOptions(options: GPURequestAdapterOption
     throw new Error('must call setDefaultRequestAdapterOptions before getGPU');
   }
   defaultRequestAdapterOptions = { ...options };
+  // MAINTENANCE_TODO: remove this once compatibilityMode is removed.
```

Comment:
I think this should be updated at the call sites outside `setDefaultRequestAdapterOptions`, rather than here. It's confusing if this function does anything other than just set results directly into `defaultRequestAdapterOptions`

Reply:
Done